lib/property_test.ex

```diff
@@ -1,30 +1,28 @@
 defmodule PropertyTest do
   @moduledoc """
   Provides macros for property testing.
-  """
-
-  # TODO: moduledoc (overview of ptesting, shrinking)
-
-  alias StreamData.Random
 
-  defmodule RunOptions do
-    @moduledoc false
+  TODO: better overview of property testing and shrinking.
+  """
 
-    defstruct test_count: 100,
-              max_shrink_depth: 50
-  end
+  # TODO: how do we customize number of tests to run and such when writing
```

Comment:
Tags? We can read the tags from the config. So we can say that `property "foo bar"` defines something like:

    property "foo bar", config

And then `check all` has access to the config. The downside of this approach is that `check all` wouldn't work inside a function. So we need to go down this road carefully. But that's already the case-ish if we import Stream.Data only inside `property`. We may want to revisit that (since Stream.Data can also be used as data factory - many we need a `next` or `random` function in Stream.Data to get a random element?)